plugins/woocommerce/includes/class-wc-post-data.php

```
@@ -621,6 +635,392 @@
+		}
+	}
+
+	/**
+	 * Hook called after a term is deleted to handle updates for product variations.
+	 *
+	 * @param int     $term_id  Term ID.
+	 * @param int     $tt_id    Term taxonomy ID.
+	 * @param string  $taxonomy Taxonomy slug.
+	 * @param WP_Term $deleted_term Copy of the already-deleted term.
+	 */
+	public static function handle_attribute_term_deleted( $term_id, $tt_id, $taxonomy, $deleted_term ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$tt_id'. (Unused Code Rules)

(UnusedFormalParameter)